### PR TITLE
fix(fastlane_session): avoid re-generating fastlane session unless required, add additional env var check

### PR DESCRIPTION
**Pull request by isaiahgrey93**
Changes:
- Stop removing the session cookie before running `spaceauth`
- Adds additional ENV var check for `CRYPTEX_PASSWORD`

### CODE REVIEW

File: lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb [hunk 1]
```diff
@@ -122,26 +122,26 @@ def self.prompt_for_version
       end
 
       # 
-      def self.create_fastlane_session()
+      def self.create_fastlane_session
```
Comment (cball) [inline lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:125]: look at you getting all ruby-like 😆

File: lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb [hunk 2]
```diff
@@ -122,26 +122,26 @@ def self.prompt_for_version
...
 
         fastlane_session_git_url =  ENV["FASTLANE_ENV_GIT_URL"]
         fastlane_session_username = ENV["FASTLANE_ENV_USERNAME"]
-
+        fastlane_session_password = ENV["CRYPTEX_PASSWORD"]
         spaceship_cookie_path = "#{File.expand_path('~')}/.fastlane/spaceship/#{fastlane_session_username}/cookie"
-        
-        if File.exists?(spaceship_cookie_path)
-          system `rm #{spaceship_cookie_path}`
-        end
```
Comment (cball) [inline lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:137]: 👍

File: lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb [hunk 3]
```diff
@@ -122,26 +122,26 @@ def self.prompt_for_version
...
 
         if !fastlane_session_username;
           UI.user_error!("No FASTLANE_ENV_USERNAME var at <root>/fastlane/.env\nFASTLANE_ENV_USERNAME is used to authenticate with the App Store for iOS releases.")
         elsif !fastlane_session_git_url;
           UI.user_error!("No FASTLANE_ENV_GIT_URL var at <root>/fastlane/.env\nFASTLANE_ENV_GIT_URL is used to store the App Store Connect session to upload releases on CI.")
+        elsif !fastlane_session_password;
```
Comment (cball) [inline lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:139]: side-note (just FYI, non-blocking) we shouldn't need semicolons here

**Review by cball — APPROVED** [review]

File: lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb (hunk 3 above)
Comment (isaiahgrey93) [reply lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:139]: HAHAHAHAHAHA

Comment (isaiahgrey93) [reply lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:139]: I can't help it

Comment (isaiahgrey93) [reply lib/fastlane/plugin/react_native_release/actions/react_native_release_action.rb:139]: Future me will remove.

**Pull request merged**